pkg/installer/generateconfig.go

```diff
@@ -150,115 +150,117 @@ func (m *manager) generateInstallConfig(ctx context.Context) (*installconfig.Ins
-					Mirrors: []string{
-						fmt.Sprintf("%s/openshift-release-dev/ocp-release-nightly", m.env.ACRDomain()),
+				Platform: types.Platform{
+					Azure: &azuretypes.Platform{
+						Region:                   strings.ToLower(m.oc.Location), // Used in k8s object names, so must pass DNS-1123 validation
+						NetworkResourceGroupName: vnetr.ResourceGroup,
+						VirtualNetwork:           vnetr.ResourceName,
+						ControlPlaneSubnet:       masterSubnetName,
+						ComputeSubnet:            workerSubnetName,
+						CloudName:                azuretypes.CloudEnvironment(m.env.Environment().Name),
+						OutboundType:             outboundType,
+						ResourceGroupName:        resourceGroup,
 					},
 				},
-				{
-					Source: "quay.io/openshift-release-dev/ocp-v4.0-art-dev",
-					Mirrors: []string{
-						fmt.Sprintf("%s/openshift-release-dev/ocp-v4.0-art-dev", m.env.ACRDomain()),
+				PullSecret: pullSecret,
+				FIPS:       m.oc.Properties.ClusterProfile.FipsValidatedModules == api.FipsValidatedModulesEnabled,
+				ImageContentSources: []types.ImageContentSource{
+					{
+						Source: "quay.io/openshift-release-dev/ocp-release",
+						Mirrors: []string{
+							fmt.Sprintf("%s/openshift-release-dev/ocp-release", m.env.ACRDomain()),
+						},
+					},
+					{
+						Source: "quay.io/openshift-release-dev/ocp-release-nightly",
+						Mirrors: []string{
+							fmt.Sprintf("%s/openshift-release-dev/ocp-release-nightly", m.env.ACRDomain()),
+						},
+					},
+					{
+						Source: "quay.io/openshift-release-dev/ocp-v4.0-art-dev",
+						Mirrors: []string{
+							fmt.Sprintf("%s/openshift-release-dev/ocp-v4.0-art-dev", m.env.ACRDomain()),
+						},
 					},
 				},
-			},
-			Publish: types.ExternalPublishingStrategy,
-			Capabilities: &types.Capabilities{
-				// don't include the baremetal capability (in the baseline default)
-				BaselineCapabilitySet: configv1.ClusterVersionCapabilitySetNone,
-				AdditionalEnabledCapabilities: []configv1.ClusterVersionCapability{
-					configv1.ClusterVersionCapabilityConsole,
-					configv1.ClusterVersionCapabilityCSISnapshot,
-					configv1.ClusterVersionCapabilityInsights,
-					configv1.ClusterVersionCapabilityMarketplace,
-					configv1.ClusterVersionCapabilityOpenShiftSamples,
-					configv1.ClusterVersionCapabilityStorage,
+				Publish: types.ExternalPublishingStrategy,
+				Capabilities: &types.Capabilities{
+					// don't include the baremetal capability (in the baseline default)
+					BaselineCapabilitySet: configv1.ClusterVersionCapabilitySetNone,
+					AdditionalEnabledCapabilities: []configv1.ClusterVersionCapability{
+						configv1.ClusterVersionCapabilityConsole,
+						configv1.ClusterVersionCapabilityCSISnapshot,
+						configv1.ClusterVersionCapabilityInsights,
+						configv1.ClusterVersionCapabilityMarketplace,
+						configv1.ClusterVersionCapabilityOpenShiftSamples,
+						configv1.ClusterVersionCapabilityStorage,
+					},
```

Comment:
are there new capabilities in 4.13 that should be enabled as well?  

`NodeTuning`?

Reply:
We can definitely, sorry that we missed it. I added to the list of enabled capabilities.